src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -317,7 +390,13 @@ export const PatientInfoTable = ({
             }}
           >
             <Button
-              onClick={() => {}}
+              onClick={() => {
+                history.replace(
```

Comment:
I think this should be `history.push`.

For reference:
push(path, [state]): (function), pushes a new entry onto the history stack
replace(path, [state]): (function), replaces the current entry on the history stack

`replace` would prevent you from accessing the previous url when you click the browser "back" button.